.github/jobs/test_install_win32.yml

```diff
@@ -41,7 +41,6 @@ jobs:
 
       # BGFX_CONFIG_MAX_FRAME_BUFFERS is set so enough Framebuffers are available before V8 starts disposing unused ones
       - script: |
-          # BGFX_CONFIG_MAX_FRAME_BUFFERS is set so enough Framebuffers are available before V8 starts disposing unused ones
```

Comment:
Why is this comment removed? Is it no longer valid? Should there be a different explanation of this define?